apps/dashboard/app/javascript/path_selector/path_selector_data_table.js

```diff
@@ -92,15 +92,22 @@ export class PathSelectorTable {
       this._table.rows.add(data.files);
       this.setLastVisited(data.path);
       this._table.draw();
-      $('#loading-icon').hide();
-      $(`#${this.tableId}_wrapper`).show();
+      this.resetTable();
     } catch (err) {
-      $('#loading-icon').hide();
-      $(`#${this.tableId}_wrapper`).show();
+      this.resetTable();
+      if (err.message.match("Permission denied")) {
+        $('#forbidden-warning').show();
+      }
       console.log(err);
     }
   }
 
+  resetTable() {
+    $('#loading-icon').hide();
+    $(this.tableWrapper).show();
```

Comment:
I haven't checked this out yet (the video appears to work), but it looks like this should be `this.tableWrapper()` - calling the function not passing it as an argument?